.ci/validation.cmd

```diff
@@ -9,20 +9,20 @@
 SET OPTION=%1
 
 if "%OPTION%" == "sevntu" (
-  call mvn -e verify -DskipTests -DskipITs -Dpmd.skip=true^
+  call mvn -ntp -e verify -DskipTests -DskipITs -Dpmd.skip=true^
     -Dspotbugs.skip=true -Djacoco.skip=true -Dxml.skip=true^
```

Comment:
https://github.com/checkstyle/checkstyle/blob/d4c9f7378d5ea6ca6cae4c832a8dc04069348341/.drone.yml#L112

Let use full name as in other places

Reply:
Done